### PR TITLE
Transaction fee calculation

### PR DESCRIPTION
**Pull request by miiiguel**
Correcting the formula for transaction fee to gasLimit times gasPrice

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
Noticed something broken while reviewing the PR so added a fix to this PR.

Thanks @miiiguel for fixing this, but in my opinion this whole paragraph is actually redundant. It seems to re-explain something we've already explained higher up in a more accessible way. @samajammin @wackerow any thoughts on this?

**Comment by wackerow** [comment]
@ryancreatescopy Ran with this a little and cleaned a few things up. I agree that paragraph had redundancy in there, and I tried to take it out. Cleaned up a bunch of other little semantic things, and also reordered the sections to flow better. 

Feedback welcome =)

File: src/content/developers/docs/gas/index.md [hunk 1]
```diff
@@ -46,13 +46,23 @@ Although a transaction includes a limit, any gas not used in a transaction is re
...
+
 ## Why can gas fees get so high? {#why-can-gas-fees-get-so-high}
 
-High gas fees are due to the popularity of Ethereum. Performing any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.
+High gas fees are due to the popularity of Ethereum. Performing any operation on Ethereum requires consuming gas, and gas space is limited per block. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, meaning each transaction takes up more space of a limited size block. This means that users must out-bid one another to be included in the next block if demand exceeds capacity by offering a higher gas price than other transactions.
```
Comment (ryancreatescopy) [inline src/content/developers/docs/gas/index.md:61]: This last sentence is too complicated: 

If there's too much demand, users must offer a higher gas price to try and out-bid other users' transactions. A higher price can make it more likely that your transaction will get into the next block.

File: src/content/developers/docs/gas/index.md [hunk 2]
```diff
@@ -46,13 +46,23 @@ Although a transaction includes a limit, any gas not used in a transaction is re
 ![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
+## What is gas limit? {#what-is-gas-limit}
+
+Gas limit refers to the maximum amount of gas you are willing to consume on a transaction. A higher gas limit means more computational work can be done while interacting with [smart contracts](/developers/docs/smart-contracts/). A standard ETH transfer requires a gas limit of 21,000 units of gas.
```
Comment (ryancreatescopy) [inline src/content/developers/docs/gas/index.md:51]: I think this paragraph is a bit confusing – specifically the second sentence... 

Try:

More complicated transactions, involving smart contracts, require more computational work so have a higher gas limit than a simple payment. A standard ETH transfer...

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

**Comment by ryancreatescopy** [comment]
@wackerow thanks just a couple tweaks needed.

**Comment by wackerow** [comment]
@all-contributors please add @miiiguel for content patch

**Pull request merged**